projects/starfish-finance/index.js

```diff
@@ -0,0 +1,27 @@
+const sdk = require("@defillama/sdk");
```

Comment:
I would recommend changing your code to below, this way we dont rely on external api:
    const { staking } = require("../helper/staking.js");

    module.exports = {
      astar: {
        tvl: () => ({}),
        staking: staking('0xa86dc743efBc24AF4c1FC5d150AaDb4DCF52c868', '0xEe8138B3bd03905cF84aFE10cCD0dCcb820eE08E', "astar", 'starfish-finance', 18)
      }
    }


I was not able to find staking contract for ethereum, you can add that part

Reply:
@g1nt0ki  Thanks. Just updated with ABIs and Coingecko API.

Currently, we don't have $SEAN staking on ETH network